.github/workflows/docker-build-and-push-dummy.yml

```
@@ -0,0 +1,69 @@
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v3
+
+      - name: Get version and Create .meta file
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:1:51: Double quote to prevent globbing and word splitting

(shellcheck)